lib/config/options/index.ts

```diff
@@ -587,6 +587,13 @@ const options: RenovateOptions[] = [
     subType: 'string',
     default: [],
   },
+  {
+    name: 'composerUpdateWithAllDependencies',
```

Comment:
@rarkins I would prefer to make this a post update option instead of introducing a new top level option.

https://docs.renovatebot.com/configuration-options/#postupdateoptions